src/pages/Recipes.js

```diff
@@ -4,6 +4,11 @@ import Footer from '../components/Footer';
 function Recipes() {
   return (
     <>
+      <header>
+        <h1 data-testid="page-title">Tela Principal de Receitas</h1>
+        <button type="button" data-testid="profile-top-btn"> Profile</button>
+        <button type="button" data-testid="search-top-btn"> Search</button>
```

Comment:
Que tal remover os espacos extras do texto?
```suggestion
        <button type="button" data-testid="profile-top-btn">Profile</button>
        <button type="button" data-testid="search-top-btn">Search</button>
```